doc/concepts/leader_election.md

```diff
@@ -0,0 +1,38 @@
+### Leading ETCD main container’s sidecar is the backup leader 
+
+- The `backup-restore sidecar` poll its own etcd main container to see if it is the leading member in the etcd cluster.
+This information is used by the backup-restore sidecars to decide that sidecar of the leading etcd main container is the `backup leader`.
+
+- Only `backup leader` sidecar among the members have the responsibility to take/upload the snapshots(full as well as incremental) for a given Etcd cluster as well as to [trigger the defragmentation](https://github.com/gardener/etcd-druid/tree/master/docs/proposals/multi-node#defragmentation) for each Etcd cluster member. 
+
+
+### Work flow
+
+Backup-restore can be in following 3 states:
+1. Follower:
+   - DefaultState for every backup-restore member is `Follower` State.
+   - If it's corresponding etcd main container is a `Follower` etcd.
```

Comment:
Replace `if it's` with `when the`

Reply:
Also, move the second point up and bring the first point of `default state` down.